docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -16,3 +16,14 @@ weight: -37
 # What’s new in Grafana Cloud
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
+
+## Support query-type template variables for Tempo data source
+
+<!-- Fabrizio Casati -->
+<!-- OSS, Enterprise -->
+
+_Generally available in Grafana Cloud_
+
+The Tempo data source now supports query-type template variables. With this, you can now create variables whose values are a list of attribute names or attribute values seen on spans received by Tempo.
+
+To learn more, refer to this [short video demo]( add link ) as well as to the [Variables page](https://grafana.com/docs/grafana/latest/dashboards/variables/) in the Grafana documentation.
```

Comment:
@fabrizio-grafana -- Once you add the link to your video demo, go ahead and merge!